### ISSUE TITLE
Current Issues for 1.11.2

### ISSUE THREAD

**Issue by djoveryde** [djoveryde/Contigo#13]
Issues for 1.11.2 to be posted here:

**Comment by djoveryde** [comment]
Might have corrected the issue unless i created a new problem. Thoughts?

What went wrong:
A problem occurred evaluating root project 'Contigo_1.11'.
Failed to apply plugin [id 'cauldron']
Value is null
Problem sits at

45 apply plugin: 'cauldron' in the build.gradle file

**Comment by CDAGaming** [comment]
@djoveryde Doing a fix that @ASCIICat Did, will test with travis if it works out

**Comment by djoveryde** [comment]
@SantaGitHub All it seems to show is that cauldrons plugin was never added to that side, im fairly sure cauldron is what links the two together

**Comment by SantaGitHub** [comment]
@djoveryde [look this repo](https://github.com/SantaGitHub/Svarka)

**Comment by djoveryde** [comment]
FAILURE: Build failed with an exception.

* Where:
Build file 'D:\Users\djove\Desktop\Contigo Tests\1.11.x\Contigo\build.gradle' line: 83

* What went wrong:
A problem occurred evaluating root project 'Contigo_1.11'.
> Could not find method minecraft() for arguments [build_6513mulkgsy5dgf3k25fd0f48$_run_closure4@1282e98] on root project 'Contigo_1.11' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

**Comment by CDAGaming** [comment]
As contigo and thermos heavily rely on cauldron, Me and Santa are now adjusting to a core with newer implementation, if youd like to help, let me know and ill add you to it

**Comment by djoveryde** [comment]
Presuming this is the one that someone else started from scratch? Would require a lot of coding to get both forge and bukkit working alongside each other

**Comment by CDAGaming** [comment]
It is https://github.com/SantaGitHub/Svarka for 1.10.2
And https://github.com/CDAGaing/Svarka for 1.12.1

**Comment by SantaGitHub** [comment]
@djoveryde yeap. It is better to switch to this Svarka than to transfer thermos, to 1.8+
Much has changed in Forge on 1.8+
Almost everything is ready in this kernel. There it remains to add events for spawning mobs from mods.

**Comment by djoveryde** [comment]
Is this actually loading the bukkit side of things as it doesn't seem to have any bukkit elements set up to load in?

**Comment by SantaGitHub** [comment]
Yes, there is Bukkit. It is simply not used as a module, but as a source code. because under 1.10.2 there are no source codes in the public domain

**Comment by CDAGaming** [comment]
@SantaGitHub All that also remains for Svarka to be builds let is to fix patch errors on both build and for genPatches

**Comment by CDAGaming** [comment]
To be buildable* Typo lol
